README.md

```diff
@@ -34,6 +39,31 @@ It is possible to send args to the script under test by adding -- after the load
 ./bin/load-runner.js -s ./scripts/dummy.js -c 5 -n 100 -r 10 -o -- -testarg1 testval1
 ```
 
+# Environment variables
+
+There are some environment variables generated for the test script that can be useful. You can use Node.js `process.env` object to access them.
+
+* `LR_FLOW_NUMBER` - number generated each run, value depends on percentages specified by `-f` parameter, see [Flow numbers](#flow-numbers)  
+* `LR_TOTAL_RUNS` - total number of runs, it's value of `-n` parameter
+* `LR_RUN_NUMBER` - current test run number
+* `LR_RAND` - randomly generated value for each run, float between 0 and 1, based on random generation with `--seed` parameter
+
+Look at [scripts/dummy_env_vars.js](scripts/dummy_env_vars.js) for example of using this.
+
+
+# Flow numbers
+
+Flow number functionality can be used inside test script to determine which branch of script to be executed. 
+It's specified by `-f` parameter and can be accessed by `LR_FLOW_NUMBER` environment variable in the script.
+
+For example this executes 5 times flow 0, 3 times flow 1, 2 times flow 2 from total of 10 runs.
+
+```
+./bin/load-runner.js -s ./scripts/dummy_flow_numbers.js -n 10 -o -f 50 30 20 --seed 123 
+```
+
+Flow numbers are randomly shuffled, but it's possible to use `--seed` parameter for repeatable result. 
```

Comment:
@kpiwko The way I understand @wojta and @david-martin's current idea, then load-runner would just be passing e.g. an A or B (or C or whatever) (or 1 or 2 or 3, etc) based on the pattern provided. Then the script will decide what it will use that for. Our script in this case will either mimick a device user or a portal user based on this flag.

This sounds like the best solution to me! :)